pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.h2gis</groupId>
     <artifactId>h2gis-geotools</artifactId>
-    <version>1.5.0</version>
+    <version>1.6.0-SNAPSHOT</version>
```

Comment:
2.0.0 because it's aligned to the H2GIS version

Reply:
Done